boaviztapi/model/consumption_profile/consumption_profile.py

```diff
@@ -31,57 +56,47 @@ class CPUConsumptionProfileModel(ConsumptionProfileModel):
     _MODEL_PARAM_NAME = ['a', 'b', 'c', 'd']
 
     def __init__(self):
-        self.cpu_manufacturer = Boattribute(
-            status=Status.NONE
-        )
-        self.cpu_model_range = Boattribute(
+        self.workloads = Boattribute(
             status=Status.NONE,
-            default=self.DEFAULT_CPU_MODEL_RANGE
+            default=self.DEFAULT_WORKLOADS,
+            unit="workload_rate:W"
         )
-        self.workloads = Boattribute(
+        self.params = Boattribute(
             status=Status.NONE,
-            default=self.DEFAULT_WORKLOADS
+            default=self._DEFAULT_MODEL_PARAMS
         )
 
     @property
     def list_workloads(self) -> Tuple[List[float], List[float]]:
-        load = [item['load'] for item in self.workloads.value]
-        power = [item['power'] for item in self.workloads.value]
+        load = [item.load for item in self.workloads.value]
+        power = [item.power for item in self.workloads.value]
```

Comment:
Okay to use object like attribute instead of a dict, though shouldn't it be `load_percentage` and `power_watt` here? According to the changes made in the DTO and in the mapper.